frameworks/kotlin/build.gradle

```diff
@@ -0,0 +1,75 @@
+buildscript {
```

Comment:
Hmmm. I might see why you decided to split this out. Prewarming one gradle build files on container start may prevent the code from building efficiently if you switch the build during code execution. 

Tricky to say. In other words, If we prewarm the image with the java gradle file, then run the code later with the kotlin, the prewarmed gradle instance may end up not being used and a new one would be used instead.